formats/application/vnd/oada/isoblue/heartbeat/1/+json/example/default.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+    _type: 'application/vnd.oada.isoblue.location.1+json',
+  
+    indexing: [
+      {
+        name: 'year-index',
+        source: 'oada.vocab.year-index',
+        value: '2019',
+      },
+      { 
+        name: 'day-index',
+        source: 'oada.vocab.day-index',
+        value: '2019-05-04',
+      },
+      {
+        name: 'hour-index',
+        source: 'oada.vocab.hour-index',
+        value: '22:00',
+      },
+    ],
+  
+    templates: {
+      '1': {
+        time: 1574056800,
```

Comment:
This is not a good template: it's just a data point.  It would make more sense to use the oada `location` vocab term which has the datum (WGS84) as part of it, and you can put that in the template.

Reply:
Fixed in 5696ced